.gitignore

```diff
@@ -24,3 +24,6 @@ yarn-error.log*
 
 backend/public
 package-lock.json
+
+__pycache__
```

Comment:
it's a dreeaaaam

Reply:
It's also even more absurd as we created this github org so that we could have multiple repos. And tbh we're basically using them as multiiple repos XD.